app/assets/js/app/modules/submitter.js

```diff
@@ -0,0 +1,25 @@
+import domready from './domready'
+import LoaderBtn from './loader-btn'
+import { trackEvent } from './analytics'
+
+domready(() => {
+  const submitBtn = new LoaderBtn('.js-btn-submit')
+  let count = 0
+  submitBtn.addEventListener('click', e => {
+    submitBtn.disable()
+
+    window.setTimeout(() => {
+      submitBtn.enable()
+      submitBtn.reset()
+
+      count++
+
+      trackEvent('send', {
+        hitType: 'event',
+        eventCategory: 'Error',
+        eventAction: 'Submission timeout',
+        eventLabel: `Attempts: ${count.toString()}`
+      })
+    }, 10000)
```

Comment:
We should use a constant for this as it's a little unclear using the value directly.